.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       run: tox
 
     - name: Run coverage
-      if: matrix.python-version == '3.12' && matrix.toxenv == 'django42'
+      if: matrix.python-version == '3.12' && (matrix.toxenv == 'django42' || matrix.toxenv == 'django52')
```

Comment:
We shouldn't run coverage twice.  Running it just on the latest django version is fine.